site/en/blog/timer-throttling/index.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Timer throttling
+authors:
+  - jakearchibald
+description: >
+  Chrome 88 will heavily throttle chained JavaScript timers for hidden pages.
```

Comment:
```suggestion
  Intensive throttling takes effect when the page has been hidden
  for more than 5 minutes, the page has been silent for at least
  30 seconds, WebRTC is not in use, and the chain of timers is
  5 or greater.
```
If you accept my suggested title change, we should also update this description because it's now redundant with the title. The description is our hint to Search about what paragraph it should surface under the title.